compiler-plugin/src/main/java/io/ballerina/stdlib/jwt/compiler/staticcodeanalyzer/JWTRule.java

```diff
@@ -0,0 +1,53 @@
+/*
+ *  Copyright (c) 2025 WSO2 LLC. (http://www.wso2.com).
+ *
+ *  WSO2 LLC. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS
+ *  OF ANY KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package io.ballerina.stdlib.jwt.compiler.staticcodeanalyzer;
+
+import io.ballerina.scan.Rule;
+
+import static io.ballerina.scan.RuleKind.VULNERABILITY;
+import static io.ballerina.stdlib.jwt.compiler.staticcodeanalyzer.RuleFactory.createRule;
+
+public enum JWTRule {
+    AVOID_WEAK_CIPHER_ALGORITHMS(createRule(
+            1,
+            "JWT should be signed and verified with "
+                    + "strong cipher algorithms",
```

Comment:
Do we need to split this into two lines?

```suggestion
            "JWT should be signed and verified with strong cipher algorithms",
```

Reply:
This doesn't exceed the 120 character limit per line.